e2e/charts/istio/istio-discovery-nacos/files/injection-template.yaml

```diff
@@ -0,0 +1,77 @@
+metadata:
```

Comment:
We just leverage Istio to help us deploy apisix-mesh-agent and apisix in the same container?

Reply:
Or we need to add a lot of things to make e2e works.

Reply:
Just install `Nacos` just like how we install `Istio`.

We don't need to use the same way to do the test, it's not the same as the Service Mesh. We can deploy APISIX and a backend container (like HTTPBIN) solely, and send requests to APISIX, as the result is expected, we then in turn proof the works are well for Nacos.